### PR TITLE
createWebSocketClient getting server id form properites file

### PR DESCRIPTION
**Pull request by Antonidass5**
I created a properites file and changed createWebSocketClient to get server id form properites file.

### CODE REVIEW

**Review by kestivvi — CHANGES_REQUESTED** [review]
I do not allow properties file to be in the `src` directory. It can be in`app` directory, it can be in the to level directory of this repo. But not in the `src`, there should be only the code. (*src* stands for *source code*)

**Comment by Vertonowsky** [comment]
Test the functions before merging. Current code is not properly loading the property value from file.

**Review by Vertonowsky — DISMISSED** [review]

File: app/src/main/java/com/example/splendormobilegame/config/Config.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        this.context = context;
+
+    }
+
+
+    private void loadFile() throws InvalidConfigException {
+        Properties properties = new Properties();
+        try {
+            InputStream inputStream = context.getAssets().open("conig.properties");
```
Comment (kestivvi) [inline app/src/main/java/com/example/splendormobilegame/config/Config.java]: I've run app at this commit. And it didn't work. I suppose this line causes errors.

**Review by kestivvi — CHANGES_REQUESTED** [review]

**Comment by kestivvi** [comment]
Please, also do the followings:
- format your code (there are for example places where there is too much blank lines)
- add java docs to the relevant parts

**Comment by kestivvi** [comment]
At the end please also resolve merge conflicts so it can actually be merged to the develop.

**Review by Vertonowsky — DISMISSED** [review]

**Review by KamilDevelopments — CHANGES_REQUESTED** [review]
You've deleted half of the repository, probadly when you were merging files you accepted your version instead of actually merging it.

**Comment by kestivvi** [comment]
I fixed it by undoing the last commit, then resolving the merge conflicts properly, and finally pushing it with git --force.

**Review by kestivvi — APPROVED** [review]

**Review by KamilDevelopments — APPROVED** [review]

**Pull request merged**